### PR TITLE
Feature/#77 update designs of microfacts tooltip

### LINKED ISSUE #77: Update desgins of Microfacts tooltip
https://user-images.githubusercontent.com/44401648/200969857-9235ca11-dddd-4707-91ad-9a498de0fdd7.png

### PR DESCRIPTION
**Pull request by DomyS**
## Related issue

Closes #77 

## Description

Added new design to microfacts tooltip.
Removed updateEntity, because it looks like we are not using it anymore. 
There is an arrow on the tooltip, should I remove it? Our article has white background, so it is not noticeable

## Screenshots (before and after the change):

microfactsTooltip

### CODE REVIEW

**Review by withchristopher — APPROVED** [review]
LGTM

File: src/components/MicrofactsTooltip/MicrofactsTooltip.tsx [hunk 1]
```diff
@@ -12,113 +12,111 @@ const useStyles = makeStyles((theme: Theme) =>
...
-            'left top',
+            'bottom right',
             'right top',
+            'top left',
+            'top right',
           ]}
-          onClose={onCloseFeedback}
-          on={['click', 'focus']}
+          closeOnDocumentClick
+          keepTooltipInside={keepTooltipInsideId}
```
Comment (tobiastykvart) [inline src/components/MicrofactsTooltip/MicrofactsTooltip.tsx:104]: I think it looks better without the arrow, you can set `arrow={false}` here.

File: src/components/MicrofactsTooltip/MicrofactsTooltip.tsx [hunk 2]
```diff
@@ -12,113 +12,111 @@ const useStyles = makeStyles((theme: Theme) =>
...
+            {description.length > 220
+              ? description.slice(0, 220) + ' ...'
+              : description}
+          </Typography>
+        </Grid>
+        <Grid item>
+          <Typography fontFamily="DM Mono" fontSize={'0.8em'}>
+            Read more on:{' '}
+            <a href={link} target="_blank" rel="noreferrer">
+              {displayLink}
```
Comment (tobiastykvart) [inline src/components/MicrofactsTooltip/MicrofactsTooltip.tsx:65]: Any idea why this is in focus when you open the tooltip? It adds that focus box around the link which draws more attention than needed.
Screenshot 2022-12-01 at 12 02 56

**Review by tobiastykvart — CHANGES_REQUESTED** [review]

File: src/components/MicrofactsTooltip/MicrofactsTooltip.tsx (hunk 1 above)
Comment (DomyS) [reply src/components/MicrofactsTooltip/MicrofactsTooltip.tsx:115]: Fixed

File: src/components/MicrofactsTooltip/MicrofactsTooltip.tsx (hunk 2 above)
Comment (DomyS) [reply src/components/MicrofactsTooltip/MicrofactsTooltip.tsx:75]: Fixed

**Review by tobiastykvart — APPROVED** [review]
LGTM

**Pull request merged**